src/tests/unit/login.test.js

```diff
@@ -135,24 +135,24 @@ describe('Testing User Login Routes with Errors', function () {
       .end((err, res) => {
         res.should.have.status(406);
         res.body.should.be.a('object');
-        res.body.should.have.property('message').eql('Password is incorrect');
+        res.body.should.have.property('message').eql('User not Found.');
```

Comment:
Why did you edit this test? I think it's supposed to test trying to log in with an incorrect password.